transports/wasm-ext/src/lib.rs

```diff
@@ -40,11 +40,17 @@ use libp2p_core::{
     transport::{ListenerId, TransportError, TransportEvent},
     Multiaddr, Transport,
 };
-use parity_send_wrapper::SendWrapper;
+use send_wrapper::SendWrapper;
 use std::{collections::VecDeque, error, fmt, io, mem, pin::Pin, task::Context, task::Poll};
 use wasm_bindgen::{prelude::*, JsCast};
 use wasm_bindgen_futures::JsFuture;
 
+#[cfg(feature = "webtransport")]
+pub mod webtransport;
+
+#[cfg(test)]
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
```

Comment:
Do you have any experience in running WASM tests in CI? It is something we are lacking at the moment which is unfortunate.

Reply:
Done :) If a crate has `wasm-bindgen-test` as a dependency then `wasm-pack test` will run for Chrome and Firefox